lambdas/package.json

```diff
@@ -1,5 +1,16 @@
 {
     "name": "di-ipv-common-lambdas",
+    "scripts": {
```

Comment:
This has been sliced out into #265 and should be removed from this PR

Reply:
This is to move the scripts section above depdendencies and make it consistent with other repos - this was not included in the smaller PR